src/cmd-coreos-prune

```diff
@@ -190,8 +190,10 @@ def main():
                                 else:
                                     print(f"No container tags to prune for build {build_id}.")
                     actions_completed.append(action)  # Append action to completed list
+        # Only add policy-cleanup for the build in builds.json if either of the action is completed
+        if actions_completed:
+            policy_cleanup = build.setdefault("policy-cleanup", {})
```

Comment:
the linter is complaining here I think the best way to fix it is to just include the for loop below inside this if condition